json/riscv_isa_spec.json

```diff
@@ -353,32 +348,41 @@
       "enabled_by": ["q"],
       "json": "isa_rv128q_addons.json"
     },
+    {
+      "xlen": [32, 64],
+      "extension": "zve64f",
+      "enabled_by": [["v", "f"]],
+      "requires": "f",
+      "enables": ["zve32f", "zve64x"]
+    },
```

Comment:
Note that this entry doesn't include a `json` field. This is allowed, and enables us to create more complicated dependency relationships without having to have an associated JSON (`zve64f` and `zve32f` have _identical_ instruction encodings as far as Mavis is concerned - they just enable different EEWs and scalar operand widths).